install/kubernetes/helm/istio/charts/egressgateway/templates/deployment.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       serviceAccountName: {{ .Values.serviceAccountName }}
       containers:
         - name: {{ template "istio.name" . }}
-          image: "{{ .Values.global.hub }}/{{ .Values.global.proxyv2.image }}:{{ .Values.global.tag }}"
+          image: "{{ .Values.global.hub }}/proxyv2:{{ .Values.global.tag }}"
```

Comment:
replacing a variable by a hardcoded string doesn't seem to go in the right direction, no ?

Reply:
yes i thought so too initially, but after discussing with @costin, we think we should not even expose proxyv2 for user to configure for the control plane as it always will be proxyv2 for 0.8.   User can only configure the data plane's envoy proxy image.

Reply:
what if they (or we) want to change the image name for other reasons
(for instance once we're done with 0.8 hopefully we'll drop the "v2" in image name)

you can use Values.global.proxy.image and have only 1 variable if you want (though I think we do need 2 because of control plane always v2 vs data plane can be v1 or v2)

Also this is for 0.8 which does support v1 no ?

Reply:
+1. Lets not hardcode the image name. If people add custom filters and want to ship their own version of proxy images (as we are seeing happening in mailing lists), we need to allow that.

Reply:
Too many options is also a problem. 

People with custom filters is a very valid use case - and the main reason we allow them to customize the sidecar image.

But the sidecar in front of pilot or mixer ? I don't see a use case or a reason to allow users to change it. 

Maybe for ingress gateway - but we can do that in 1.0

Reply:
call the variable proxy-control-plane or similar?